main_app/templates/items/detail.html

```diff
@@ -27,26 +25,46 @@ <h1>Item Details</h1>
   </div>
 </div>
 
-<hr>
 
+<hr />
```

Comment:
We might need to discuss if we should make this practice universal within the code base